storage/client/client.go

```diff
@@ -1269,6 +1272,22 @@ func (c *StorageClient) RuntimeEvents(ctx context.Context, p apiTypes.GetRuntime
 		h := ethCommon.HexToHash(*p.EvmLogSignature)
 		evmLogSignature = &h
 	}
+	var evmLogContractECAddr *ethCommon.Address
+	if p.EvmLogContractAddrEth != nil {
+		evmLogContractECAddr = common.Ptr(ethCommon.HexToAddress(*p.EvmLogContractAddrEth))
+	}
+	var erc721TransferTokenIdB64 *string
+	if p.Erc721TransferTokenId != nil {
+		erc721TransferTokenIdBI := &big.Int{}
+		if err := erc721TransferTokenIdBI.UnmarshalText([]byte(*p.Erc721TransferTokenId)); err != nil {
+			return nil, fmt.Errorf("unmarshalling erc721_transfer_token_id: %w", err)
+		}
+		erc721TransferTokenIdBuf, err := abi.Arguments{evmabi.ERC721.Events["Transfer"].Inputs[2]}.Pack(erc721TransferTokenIdBI)
+		if err != nil {
+			return nil, fmt.Errorf("ABI-packing erc721_transfer_token_id: %w", err)
+		}
+		erc721TransferTokenIdB64 = common.Ptr(base64.StdEncoding.EncodeToString(erc721TransferTokenIdBuf))
+	}
```

Comment:
yeah, it is more consistent with the project's style to read that into new columns. let's not do that in this PR though

Reply:
OK. It will be a fair bit more work to refactor this later than to do it right now I think, especially because of the reindex. But fair enough to push to get this PR (and feature) out.

Reply:
gonna file an issue and resolve this thread

Reply:
#601